jenkins/macOS.groovy

```diff
@@ -75,14 +75,58 @@ node('built-in') {
   }
 
   stage('Archive build artifacts') {
+    // TODO (mac auto-update v2):
+    // This Jenkins step manually generates latest-mac.yml and uploads artifacts to wire-taco.
+    // Once macOS wrapper builds are moved to electron-builder, drop this and use
+    // `electron-builder --publish` to create + upload the update metadata instead.
+
     if (!production && !custom) {
       // Internal
-      sh "ditto -c -k --sequesterRsrc --keepParent \"${WORKSPACE}/wrap/build/WireInternal-mas-universal/WireInternal.app/\" \"${WORKSPACE}/wrap/dist/WireInternal.zip\""
+      def appPath = "${WORKSPACE}/wrap/build/WireInternal-mas-universal/WireInternal.app/"
```

Comment:
```suggestion
      def appPath = "${WORKSPACE}/wrap/build/WireInternal-mac-universal/WireInternal.app/"
```

Reply:
I think it is a typo. It needs to be fixed at 2 other places too if you are fixing this.